docs/FeatureManagement/Clients/SpringCloud/Spring.FeatureFlag.v1.0.0.schema.json

```diff
@@ -0,0 +1,89 @@
+{
+    "definitions": {},
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://azconfig.io/schemas/FeatureManagement/v1.0.0/SpringFeatureFlag.json",
+    "type": "object",
+    "title": "An Spring Cloud Azure App Configuration Feature Declaration",
```

Comment:
It's the feature management section of the Spring Cloud configuration. Not a feature declaration.